src/Teams/Repository/TeamMemberRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Data;
+using Teams.Models;
+
+namespace Teams.Repository
+{
+    public class TeamMemberRepository:IRepository<TeamMember>
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public TeamMemberRepository(ApplicationDbContext dbContext)
+        {
+            this._dbContext = dbContext;
+        }
+
+        public async Task<TeamMember> DeleteAsync(TeamMember entity)
+        {
+            var obj=_dbContext.TeamMembers.Remove(entity);
+            await _dbContext.SaveChangesAsync();
+            return obj.Entity;
+        }
+
+        public IQueryable<TeamMember> GetAll()
+        {
+            return _dbContext.TeamMembers;
+        }
+
+        public IQueryable<TeamMember> GetById(TeamMember Id)//public async Task<TeamMember> GetById(int Id) => await _dbContext.TeamMember.FindAsync(Id);
```

Comment:
Please fix this method as it according to the naming convention should return exactly one entity base on identifier. It should look like TeamMember GetById(<TypeOfId> id). No need to return IQueryable as there is no more restricted constrains than Id that can exist